spec/app/domain/authentication/authenticate_spec.rb

```diff
@@ -94,8 +75,11 @@ def authenticator(pass:)
       )
 
       Authentication::Authenticate.new(
-        token_factory: token_factory,
-        audit_log: audit_logger
+        validate_webservice_access:      mock_validate_webservice_access(validation_succeeded: true),
+        validate_whitelisted_webservice: mock_validate_whitelisted_webservice(validation_succeeded: true),
+        validate_origin:                 mocked_origin_validator,
+        token_factory:                   token_factory,
```

Comment:
The token factory is not mocked in this case?

Reply:
renamed